src/components/header/_header.scss

```diff
@@ -101,14 +101,16 @@
 
     &:link,
     &:visited {
-      margin-bottom: -1px; // Negate transparent bottom border
-      border-bottom: 1px solid transparent;
       text-decoration: none;
     }
 
     &:hover,
     &:active {
-      border-bottom-color: currentColor;
+      margin-bottom: -1px; // Negate transparent bottom border
+      border-bottom: 1px solid currentColor;
+      @include govuk-if-ie8 {
```

Comment:
could you include a comment that explains why the ie8 code is needed? the info in the PR is very good